.github/workflows/testsigma-cicd.yml

```
@@ -14,6 +14,7 @@ jobs:
   build:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)